crates/re_query_cache/src/latest_at.rs

```diff
@@ -96,6 +96,11 @@ impl SizeBytes for LatestAtCache {
 }
 
 impl LatestAtCache {
+    #[inline]
+    pub fn is_cached(&self, query: &LatestAtQuery) -> bool {
```

Comment:
who's actually using this? / what's the envisioned usecase?

Reply:
removed